### PR TITLE
SUBMARINE-423. Update the E2E test document

### PR DESCRIPTION
**Pull request by kevin85421**
### What is this PR for?
Update the E2E test document

### What type of PR is it?
[Improvement]

### What is the Jira issue?
https://issues.apache.org/jira/browse/SUBMARINE-423#

### How should this be tested?
https://travis-ci.org/github/kevin85421/hadoop-submarine?utm_medium=notification&utm_source=github_status

### Questions:
* Does the licenses files need an update? No
* Are there breaking changes for older versions? No
* Does this needs documentation? No

### CODE REVIEW

File: docs/development/IntegrationTest.md [hunk 1]
```diff
@@ -12,17 +12,13 @@
...
+2.  Currently, there are some differences between `test-e2e` and `test-k8s` in operation mode. To elaborate, `test-e2e` needs to deploy Apache Submarine locally, while `test-k8s` deploys Apache Submarine via k8s.
 
-They are in the project's `submarine/submarine-test` directory, There are two modules, `e2e` and `test-k8s`.
-
-There are currently some differences between `test-e2e` and `test-k8s` in operation mode.
-
-Among them, `test-e2e` needs to deploy submarine locally, while `test-k8s` uses k8s to deploy submarine.
-
-These two different test methods can be applied to different test scenarios. (In the future, these two test methods may be combined or adjusted)
+3.  These two different test methods can be applied to different test scenarios. (In the future, these two test methods may be combined or adjusted)
```
Comment (yuanzac) [inline docs/development/IntegrationTest.md]: How about we change two methods to two test modules

**Comment by pingsutw** [comment]
Thanks @kevin85421, GitHub actions failure seems unrelated

File: docs/development/IntegrationTest.md [hunk 2]
```diff
@@ -12,17 +12,13 @@
   limitations under the License. See accompanying LICENSE file.
 -->
 
-# IntegrationTest
+# Integration Test
+## Introduction
+1.  Now, Apache Submarine supports two kinds of integration tests, `test-e2e` and `test-k8s`, and these two modules can be found in the [submarine/submarine-test](https://github.com/apache/submarine/tree/master/submarine-test) directory.
```
[the remark this answers is not in the document]
Comment (kevin85421) [reply docs/development/IntegrationTest.md]: @yuanzac Thank you for your review. I have fixed the problem.

File: docs/development/IntegrationTest.md (hunk 1 above)
Comment (kevin85421) [reply docs/development/IntegrationTest.md]: Thank you for your review. I have changed "test methods" to "test modules".

**Pull request closed without merging**